### DISCUSSION TITLE
Source Google Analytics Data API (GA4): Issue with Updating/Deleting Custom Reports

### DISCUSSION THREAD

**Discussion by gverdugo-swo** [airbytehq/airbyte#32822] [General]
Hello,

I'm encountering an issue while attempting to update or delete custom reports from my GA4 source. Each time I attempt to interact with the HTML elements, either by clicking or hovering over them, I receive an error message stating "Oops! Something went wrong…".

![image](https://github.com/airbytehq/airbyte/assets/148429304/6c605d69-eb8e-4ad1-90a7-00368459caa3)

To troubleshoot, I've tried accessing the platform using both Microsoft Edge and Google Chrome browsers, but the issue persists. Additionally, I've observed an error in the browser, which may be pertinent to resolving this problem. The error log can be found here:

[localhost-1701098094498.log](https://github.com/airbytehq/airbyte/files/13476849/localhost-1701098094498.log)

Any assistance in resolving this matter would be greatly appreciated.

**Comment by marcosmarxm** [comment]
Can you try to get the connector configuration using the Airbyte API? 
What version of the connector are you using today? Any chance you have upgraded?

**Reply by gverdugo-swo** [reply]
### Can you try to get the connector configuration using the Airbyte API?
I don't why this would help, I want to delete a custom report, and I can't do that in an on-premise instance.

### Can you try to get the connector configuration using the Airbyte API?
I am using v2.0.3.

I upgraded the connector a few weeks ago, and the problem existed at that time.

**Reply by marcosmarxm** [reply]
Do you what version you're using before the upgrade? (I'll try to replicate the issue in my side but need this info to be able to do it).
I'm asking the API response because I want to check how the database is storing the connector config, from what you're displaying it looks there are some problems with the custom report object.

**Reply by gverdugo-swo** [reply]
Maybe the 2.0.2, I don't really know...

[a comment by an automated account is not shown]
